maestro-driver-ios/maestro-driver-iosUITests/Routes/Handlers/ScreenDiffHandler.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import XCTest
+import CryptoKit
+import FlyingFox
+import os
+
+class ScreenDiffHandler : RouteHandler {
```

Comment:
We already have screenshot comparison logic within Maestro (that also allows to configure threshold), can we reuse that one instead of rebuilding it in Swift?

Reply:
Had some issues with image-comparison library there, even when sampling / resizing images it wasn't performant enough comparing to this approach, hash function in Swift is much faster.

Reply:
Ok, let’s discuss this in person, I doubt that the performance difference is that large. The reason we are moving to screneshot comparison is to eliminate delay in viewHierarchy calls, so perhaps the compression speed is not really an important factor here?